lib/Runtime/PlatformAgnostic/RuntimePlatformAgnosticPch.h

```diff
@@ -15,6 +17,42 @@
 #pragma warning(pop)
 #endif
 
+// Minimal definitions to use AssertMsg in the PAIL
+#ifndef _WIN32
+#define DbgRaiseAssertionFailure() __builtin_trap()
+#define __analysis_assume(x)
+#define __in
+#define __inout
+
+#ifndef TRUE
```

Comment:
Do we need `TRUE`/`FALSE` on Linux? Maybe they are only used on `Windows` implementation?